documentation/architecture.md

```diff
@@ -191,7 +191,7 @@ A simplified format in JSON can be used to represent rules. The former visual ru
 ```json
 {
 	"name" : "prueba-test",
-	"text" : "select *,\"prueba-test\" as ruleName  from pattern [every ev=iotEvent((cast(id?, String)  regexp  \"asd\"))]",
+	"text" : "select * from pattern [every ev=iotEvent((cast(id?, String)  regexp  \"asd\"))]",
```

Comment:
Doubt: at rule creation time (`POST /rules`, if I'm remembering correctly) it is clear that ruleName is not include in the EPL and it is automatically added.

However, what about of `GET /rules/:id`? Is would be the automatically added ruleName added there?

Reply:
Yes. The ruleName is added during the creation automatically in the 'text' field. If the rule is requested using GET / rules /: id or / rules, it will include the ruleName in the 'text' field.

Reply:
Fixed in 0917cbd